internal/dataplane/deckgen/generate.go

```diff
@@ -121,6 +121,15 @@ func ToDeckContent(
 		return strings.Compare(*content.CACertificates[i].Cert, *content.CACertificates[j].Cert) > 0
 	})
 
+	for _, c := range k8sState.Licenses {
+		content.Licenses = append(content.Licenses,
+			file.FLicense{License: c})
+	}
+
+	sort.SliceStable(content.CACertificates, func(i, j int) bool {
+		return strings.Compare(*content.CACertificates[i].Cert, *content.CACertificates[j].Cert) > 0
+	})
```

Comment:
Maybe the intention is to sort `Licenses` here?
```suggestion

	sort.SliceStable(content.CACertificates, func(i, j int) bool {
		return strings.Compare(*content.Licenses[i].License, *content.CACertificates[j].License) > 0
	})
```

Reply:
Yeah, missed part of the range replace after copy/paste, and changing the field.

We shouldn't ever _have_ more than one license, but it is technically a list, so keeping the (fixed) sort in case we ever decide to make use of the additional slots.